.github/workflows/build-binaries.yml

```diff
@@ -42,7 +42,6 @@ jobs:
       - name: Get commit short hash
         run: node ./scripts/get-commit-short-hash.mjs
       - name: Build
-        working-directory: ./packages/cli/core
```

Comment:
ergo, will run from the root of the repo, thus triggering `nx` and the topological run